ui/src/app/pages/workspace/workspace-card.tsx

```diff
@@ -271,16 +292,18 @@ export const WorkspaceCard = fp.flow(withNavigation)(
                           )
                         }
                       >
-                        <div
+                        <StyledRouterLink
```

Comment:
This is why some of the tests are failing - because this is here, the workspace card test component needs to be wrapped in a `<MemoryRouter>`